functions-and-operators/string-functions.md

```diff
@@ -163,11 +163,71 @@ Insert a substring at the specified position up to the specified number of chara
 
 ### [`INSTR()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_instr)
 
-Return the index of the first occurrence of substring.
+The `INSTR(str, substr)` function is used to get the index of the first occurrence of the second given argument `substr` in the first given argument `str`. Each argument can be either a string or a number. This function is the same as the two-argument version of [`LOCATE(substr, str)`](/functions-and-operators/string-functions.md#locate), but with the order of the arguments reversed.
+
+> **Note:**
+>
+> `INSTR(str, substr)` is case-sensitive by default, since [TiDB defaults to using binary collations](/character-set-and-collation.md), which differs from MySQL.
+
+- If either argument is a number, the function treats the number as a string.
+- If `substr` is not in `str`, the function returns 0; otherwise, it returns the index of the first occurrence of `substr` in `str`.
```

Comment:
```suggestion
- If `substr` is not in `str`, the function returns `0`. Otherwise, it returns the position of the first occurrence of `substr` in `str`.
```